doc/user/installation.rst

```diff
@@ -103,6 +117,51 @@ options from the list below.
 
    Do not build bgpd.
 
+.. option:: --disable-ldpd
+
+   Do not build ldpd.
+
+.. option:: --disable-nhrpd
+
+   Do not build nhrpd
+
+.. option:: --disable-eigrpd
+
+   Do not build eigrpd.
+
+.. option:: --disable-babeld
+
+   Do not build babeld.
+
+.. option:: --disable-watchfrr
+
+   Do not build watchfrr.  Watchfrr is used to integrate daemons into startup/shutdown
+   software available on your machine.  This is needed for systemd integration.
```

Comment:
Last sentence should be clearer; it could be read to mean that `--disable-watchfrr` is needed for systemd integration.

Reply:
fixed